releasetasks/templates/firefox/bouncer_aliases.yml.tmpl

```diff
@@ -0,0 +1,45 @@
+{% set buildername = "release-{}-{}_bouncer_aliases".format(branch, product) %}
+{% include "post_release_human_decision_task.yml.tmpl" %}
```

Comment:
I see this template is included twice, here and in `version_bump.yml.tmpl`. Not sure how the tests pass, but in real world situation you will end up with 2 same tasks with same task ID.

I'd probably include `post_release_human_decision_task.yml.tmpl` from `release_graph.yml.tmpl` using something like `if postrelease_bouncer_aliases_enabled or postrelease_version_bump_enabled or ...`.

This makes me think that we should add a test to generate a graph with all features enabled and run common tests.